lumigator/python/mzai/backend/backend/tests/conftest.py

```diff
@@ -107,40 +107,45 @@ def db_session(db_engine: Engine):
             session.rollback()
 
 
-@pytest.fixture(scope="session")
-def localstack_container():
-    """Initialize a LocalStack test container."""
-    with LocalStackContainer("localstack/localstack:3.4.0") as localstack:
-        yield localstack
+@pytest.fixture(scope="function")
+def fake_s3fs() -> S3FileSystem:
+    # ...and patch the s3fs name with the Fake
```

Comment:
let's make sure method comments follow docstring syntax like on the left

Reply:
Ok, I'll add some comments on the fixtures.